ftplugin/rust.vim

```diff
@@ -106,6 +106,9 @@ command! -nargs=* -bar -buffer RustEmitIr call rust#Emit("ir", [<f-args>])
 " See |:RustEmitAsm| for docs
 command! -nargs=* -bar -buffer RustEmitAsm call rust#Emit("asm", [<f-args>])
 
+" See |:RustPlay| for docs
+command! -range=% RustPlay :call rust#Play(<count>, <line1>, <line2>, <f-args>)
```

Comment:
You need to add the appropriate `delcommand` to `b:undo_ftplugin`. See [line 144](#diff-9f6aaa9ba985bf46731062ecf337d32aR144).